examples/lighting-app/qpg/BUILD.gn

```diff
@@ -25,7 +25,7 @@ import("${qpg_sdk_build_root}/qpg_sdk.gni")
 # declares chip_build_libshell = false
 import("${chip_root}/src/lib/lib.gni")
 
-# declares chip_enable_pw_rpc = false
+# declares chip_enable_pw_rpexamples/lighting-app/qpg/src/c = false
```

Comment:
This seems like a seach and replace typo

Reply:
```suggestion
# declares chip_enable_pw_rpc = false
```